doc/toolbox-create.1.md

```diff
@@ -82,7 +82,8 @@ confusion.
 **--distro** DISTRO, **-d** DISTRO
 
 Create a toolbox container for a different operating system DISTRO than the
-host. Cannot be used with `--image`.
+host. Cannot be used with `--image`. Has to be coupled with `--release` unless
+the selected system matches the host.
```

Comment:
Nitpick: your commit message reminded me that *selected DISTRO* would be better, because it's a good opportunity to refer to the argument as we call it elsewhere.

Reply:
This was fixed, and merged through https://github.com/containers/toolbox/pull/986